acj/core.py

```diff
@@ -27,6 +28,9 @@
 # create custom namespace for signals
 event = Namespace()
 
+# store celery variable
+celery = Celery()
```

Comment:
Looks like you are create a default celery object here. And tasks is using this celery instead of the ones setup in the create app function.